packages/plugin-manual-notification/src/server/actions/notificationsController.ts

```
@@ -0,0 +1,34 @@
+      const { title = '', detail: content = '', level = 'open', duration = null, notifyType } = ctx.action.params;
+      switch (notifyType) {
+        case NoticeType.MODAL:
+          ctx.app.noticeManager.modal(title, content, level);
+          break;
+        case NoticeType.STATUS:
+          ctx.app.noticeManager.status(content, level, duration);
+          break;
+        case NoticeType.TOAST:
+          ctx.app.noticeManager.toast(content, level);
+          break;
+        case NoticeType.NOTIFICATION:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)